### ISSUE TITLE
Webpack 3.10.0 incompatibility

### ISSUE THREAD

**Issue by montogeek** [getsentry/sentry-webpack-plugin#28]
Hello again!
Webpack 3.10.0 allows you to pass an array of rules using the `rules` or `loaders` key: 
```js
module: {
	loaders: [
	  {
	    test: /\.handlebars$/,
	    exclude: /(node_modules|bower_components|public\/)/,
	    use: [
	      {
	        loader: 'handlebars-loader',
	        options: { inlineRequires: 'static/icons' }
	      }
	    ]
	  },
	]
}
```
But this plugin only checks for the `rules` key, since it is undefined, it would reset the rules, deleting the user loaders configuration:
https://github.com/getsentry/sentry-webpack-plugin/blob/955615b3d5ba9bdf42dfbaa065443562594569f3/src/index.js#L38-L40

I guess webpack supports this for historical reasons, I am not sure if the plugin should handle this case or the user should be more careful with its webpack configuration or webpack should no support it.

More info in the webpack migration guide: https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules
Source code on webpack side: https://github.com/webpack/webpack/blob/f352436a0fd6491b8cc1aadeddfbe02c878686ff/lib/NormalModuleFactory.js#L49

As you can see `rules` is used if defined.

This bug was introduced in https://github.com/getsentry/sentry-webpack-plugin/pull/20

Since it is supported by webpack I think the plugin should support it.

The solution should be adding an additional check for `loaders` key.

**Comment by kamilogorek** [comment]
Fix published as `1.3.3`. Please let us know if it solves the issue :)

**Comment by montogeek** [comment]
It works, thank you!
